worldengine/simulations/basic.py

```diff
@@ -1,14 +1,15 @@
-def find_threshold(elevation, land_percentage, ocean=None):
-    width = len(elevation[0])
-    height = len(elevation)
+import numpy
+
+def find_threshold(map, land_percentage, ocean=None):#never used anywhere?
+    height, width = map.shape
```

Comment:
can we get away from using `map`? rename back to elevation or use something generic like map_data or data?

map is a keyword: https://docs.python.org/2/library/functions.html#map